azerothcore/azerothcore-wotlk
# fix(DB/creature_template):  Fix Twilight Fissure interaction

This change will prevent the chance of the Twilight Fissure to not go in accordance with the visual=damage





## CHANGES PROPOSED:
-  Add unit_flag 33554432
-  Add ScriptName "trigger_periodic" as other zone effects triggers


## ISSUES ADDRESSED:
- There are some times where the fissure can follow for a little bit a layer so the damage could not go in accordance with the actual effect.
![image](https://user-images.githubusercontent.com/61223313/97792195-96950d00-1ba0-11eb-8238-587a5c8c724a.png)




## TESTS PERFORMED:
- Tested in-game



## HOW TO TEST THE CHANGES:


Apply SQL on db
Go fight with Vesperon as Sartharion is currently broken on last rev, see #3633 
No matter how many encounters yo do the fissure is static and the damage goes in accordance with the visual zone


## KNOWN ISSUES AND TODO LIST:



## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
